app/controllers/temp/tracks/exercises_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Temp
```

Comment:
This is a temporary controller which responds to when the user hits "Start exercise" on the iteration zero state.